### DISCUSSION TITLE
Package build differences on Mac (Big Sur)

### DISCUSSION THREAD

**Discussion by lavabyrd** [soundshed/soundshed-app#19] [General]
Hey Chris,

I had a few differences trying to get this running locally and ended up needing to add

``` "electron-prebuilt-compile": "8.2.0", ``` to the package.json 

An update... So this one nearly broke my heart. It builds fine. I just had a space in my paths that the `npm run start` really didn't like. 

e.g. `/Personal Code/soundshed...` will throw errors but `/PersonalCode/soundshed...`  doesn't. 🤦🏻 for me but at least we have the history of it here if it happens to anyone else

**Comment by webprofusion-chrisc** [comment]
Ah, well good to know :) - the Mac build works automatically using GitHub actions on a cloud hosted macOS, not sure what the latest macOS version they use is but I assume it's Big Sur.
